.golangci.yml

```diff
@@ -59,20 +57,28 @@ linters:
     - varcheck
 
 linters-settings:
-  # Show all issues from a linter.
-  max-issues-per-linter: 0
-
-  # Show all issues with the same text.
-  max-same-issues: 0
```

Comment:
Should be under `issues` section according to https://github.com/golangci/golangci-lint/blob/9b20d49dc234cae4819cea4cfd64d0cc507310c4/.golangci.reference.yml#L2569C1-L2577C1